src/polycap.c

```diff
@@ -39,6 +39,12 @@
 #endif
 #include <complex.h> //complex numbers required for Fresnel equation (reflect)
 
+#ifndef _POLIFITGSL_H //used in describing polycapillary shape
```

Comment:
Die `#ifndef`, `#define` en `#endif` mogen weg trouwens aangezien `_POLIFITGSL_H` toch nooit gedefinieerd zal zijn.

Reply:
True.
Hou ik die fit procedure trouwens mss niet beter als aparte library file? Voor de schoonheid van de zaken? Of maakt het niet zoveel uit en is het goed zoals het nu is?

Reply:
Als dit de enige plaats is waar je er gebruik van zal maken, kan je het even goed hier houden.